app/presenters/base_letter_presenter.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class BaseLetterPresenter < BasePresenter
+  def contact_full_name
+    format_name(contact_first_name, contact_last_name)
+  end
+
+  def date_of_letter
+    Time.now.to_date.to_formatted_s(:day_month_year)
+  end
+
+  def human_business_type
+    I18n.t("waste_exemptions_engine.pdfs.certificate.busness_types.#{business_type}")
+  end
+
+  def operator_address_one_liner
+    address_lines(operator_address).join(", ")
+  end
+
+  def site_address_one_liner
+    address_lines(site_address).join(", ")
+  end
+
+  private
+
+  def format_name(first_name, last_name)
```

Comment:
What about including the `waste_exemptions_engine/application_helper` here and use the implementation of this method from it?

Reply:
Resolved by https://github.com/DEFRA/waste-exemptions-back-office-ta/pull/357/commits/75412f2a638796b5192063b3ebd8505c92f459b2